### PR TITLE
Add Retrieve Workflow Activity Records microflow action

### PR DESCRIPTION
**Pull request by DmytroKost**
For 10.5.0

### CODE REVIEW

File: static/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records/properties.png
Comment (Yiyun333) [inline static/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records/properties.png:1]: @DmytroKost This screenshot is not correct. It shows the properties of **Retrieve workflows**, not **Retrieve workflow activity records**. We need to replace this with a correct screenshot.

**Comment by Yiyun333** [comment]
This feature didn't make it in 10.5, will be released in 10.6. Info from Dmytro via Slack.

File: static/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records/properties.png
Comment (DmytroKost) [reply static/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records/properties.png:1]: Yeah, you're right. I will correct it.

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+This activity can only be used in **Microflows**.
+{{% /alert %}}
+
+## 1 Introduction
+
+A **Retrieve workflow activity records** activity can be used to retrieve a list of **System.WorkflowActivityRecord** objects sorted by activity start time.
+
+It needs the **System.Workflow** instance object as its input. It will break at Runtime if the passed workflow object is empty.
+
+The **System.WorkflowActivityRecord** entity represents a snapshot of workflow activity attributes taken at the moment when the activity finishes or the current attribute values if it is still in progress. The most common use case for the retrieved list of **System.WorkflowActivityRecord** objects is to build an activity timeline for a specific workflow. Each **System.WorkflowActivityRecord** object is linked to the previously executed activity through the **System.WorkflowActivityRecord_PreviousActivity** association (except for the Start event). It can be used to visualize the order in which each activity finishes.
```
Comment (PaulJongenMx) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records.md]: THe first line is correct but had to read it 2x. Maybe split it in 2 sentences?

File: static/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records/properties.png
Comment (DmytroKost) [reply static/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records/properties.png:1]: Replaced the screenshot

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records.md (hunk 1 above)
Comment (DmytroKost) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/workflow-activities/retrieve-workflow-activity-records.md]: Split it in several sentences

**Review by Yiyun333 — APPROVED** [review]

**Pull request merged**